docs/get-started/quick-start-guide.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Quick Start
+---
+
+If you’re a Python developer, this Planet SDK for Python makes it easy to access Planet’s massive repository of satellite imagery and add Planet data to your data ops workflow.
+
+If you’re not a Python developer, you can use the Command Line Interface (CLI) to get Planet data, and to process and analyze that data.
+
+Take the following steps to install the SDK and connect with the Planet Server.
+
+[TOC]
+
+## Step 1: Install Python 3.7+ and a virtual environment
+
+This is a Python package, so you’ll need to install Python (version 3.7 or greater), and setup and install a virtual environment.
+
+Yes. Even if you’re not writing code—and only using the "no code" CLI part of the Planet SDK for Python—you’re using Python to communicate with the Planet Labs PBC servers. It’s not too tricky, but it does require a presence of mind to complete. If you need help with Python install and setting up a virtual environment, read [Get started with venv & Planet SDK](venv-tutorial.md).
+
+## Step 2: Install the Planet SDK for Python
+
+Install the Planet SDK for Python using [pip](https://pip.pypa.io):
+
+```console
+$ pip install planet --pre --user
+```
+
+The [--user](https://pip.pypa.io/en/stable/user_guide/#user-installs) flag ensures the Python packages are installed relative to your user home folder. It is recommended for those new to pip.
+
+## Step 3: Check the Planet SDK for Python version
+
+```console
+$ planet --version
+```
+
+You should be on some version 2 of the Planet SDK for Python. If you’re not, or you need to 
+
+## Step 4: Sign on to your account
+
+Planet SDK for Python, like the Planet APIs, require an account for use.
+
+### Have your Planet account user name and password ready
+
+To confirm your Planet account, or to get one if you don’t already have one, see [Get your Planet Account](get-your-planet-account.md).
+
+### Authenticate with the Planet server
+
+Just as you log in when you browse to https://account.planet.com, you’ll want to sign on to your account so you have access to your account and orders.
+
+At a terminal console, type the following Planet command:
+
+```console
+$ planet auth init
+```
+
+You’ll be prompted for the email and password you use to access [your account](https://account.planet.com). When you type in your password, you won’t see any indication that the characters are being accepted. But when you hit enter, you’ll know that you’ve succeeded because you’ll see on the command line:
+
+```console
+Initialized
+```
+
+### Get your API key
```

Comment:
Here, we should probably discuss the `PL_API_KEY` environment variable that will override authorization stored by `planet init` if it exists.

Reply:
This is now ticketed in #749, so resolving this conversation.